Assets/HotmaxEditorVR/common/stateManager.cs

```diff
@@ -9,11 +9,22 @@ public enum editorModes
     {
         universalTransformMode,
         cloneDeleteMode,
-        openMenuMode
+        openMenuMode,
     }
     public delegate void editorModeHandler(editorModes value);
     public static event editorModeHandler editorModeEvent;
 
+    public static activeQuadrants activeQuadrant = activeQuadrants.quarant0;
```

Comment:
you arent actually using these enums -- since you are just passing an int through the mutator

Reply:
i would probably use an enum though in your code -- but make it `TOP RIGHT LEFT BOTTOM` -- you'll have to update inputManager.cs and editorModeUIController to set/accept enum values, as well as your mutator below